platform/vmaas.go

```diff
@@ -22,14 +22,18 @@ func updatesHandler(c *gin.Context) {
 						"erratum": "RH-1",
 						"package": "firefox-0:77.0.1-1.fc31.x86_64",
 						"releasever": "ser1",
-						"repository": "repo1"
+						"repository": "repo1",
+                        "package_name": "firefox",
```

Comment:
Just a formatting nitpick - this is indented by spaces while old lines by tabs.

Reply:
fixed